components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/resources/devportal-api.yaml

```
@@ -871,29 +871,1082 @@ paths:
+        $ref: docs/examples/apis/comment_id-comments.yaml#/get
+
+  /apis/{apiId}/topics:
+    get:
+      tags:
+        - Topics
+      summary: |
+        Get a list of available topics for a given async API
+      description: |
+        This operation will provide a list of topics available for a given Async API, based on the provided API ID.
+      parameters:
+      - $ref: '#/components/parameters/apiId'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)